ksql-parser/src/test/java/io/confluent/ksql/parser/SqlFormatterTest.java

```diff
@@ -148,5 +222,18 @@ public void shouldFormatOuterJoinWithoutJoinWindow() {
     final String expected = "left L\nFULL OUTER JOIN right R ON (('left.col0' = 'right.col0'))";
     assertEquals(expected, SqlFormatter.formatSql(join));
   }
+
+  @Test
+  public void shouldFormatSelectQueryCorrectly() {
+    final String statementString =
+        "CREATE STREAM S AS SELECT a.address:city FROM address a;";
+    final Statement statement = KSQL_PARSER.buildAst(statementString, metaStore).get(0);
+
+    String s = SqlFormatter.formatSql(statement);
+
+    System.out.println();
```

Comment:
What is this testing exactly?

Reply:
Oops, I forgot to complete this one. Updated it.